packages/aws-rfdk/lib/lambdas/nodejs/lib/configure-spot-event-plugin/spot-event-plugin-client.ts

```diff
@@ -94,6 +118,47 @@ export class SpotEventPluginClient {
     }
   }
 
+  public async getCollection(type: CollectionType): Promise<string[]> {
```

Comment:
A couple of observations/suggestions, but they're not necessary changes:

Here, we're assuming Deadline collections are the same type (in this case `string[]`). This may not always be the case, but it should probably be fine for now seeing as there are only `/db/settings/collections/<type>`  endpoints for groups/pools.

Another thing I want to point out is that we might want to consider returning `Promise<PoolCollections>` from this method rather than just returning the value of the `.Pools` field. If callers of this method only happen to need the `.Pools` field, they can extract it themselves, but since this class is specifically for SEP, it's probably fine that we make the assumption that the `.ObsoletePools` field is not needed.